tests/framework/test_engine.py

```diff
@@ -347,6 +348,17 @@ def test_SimulationContext_report_write(SimulationContext, base_config, componen
         assert results.equals(written_results)
 
 
+@pytest.mark.skip(reason="TODO: Figure out how to make Dill serialize in pytest")
```

Comment:
Could you mock dill to get the tests at least to run?

Reply:
done